Dockerfile

```diff
@@ -1,8 +1,12 @@
 FROM moexmen/node:8
```

Comment:
Do we need to use any functionality specific to `moexmen/node`?

Reply:
not really, any node 8 will do

Reply:
Nope, but `moexmen/node` has some standardization across all the docker images that we are using. Such as timezone configuration and a `WORKDIR` setting.

Coming to think of it, I'll make a change to copy the files to the `/app` folder instead.

Reply:
oh I didn't now the name of the folder mattered.

Reply:
It didn't, but it's more of a standardization. So we know that if we enter a container built by us, we know that `/app` folder will continue the application code.

Reply:
What do you think about using the base directory instead of `/app`? It seems clearer and more obvious to me.

Reply:
What do you mean by base directory?

Reply:
The first directory you see when you enter the container.

Reply:
Ah yes. `ADD . .` will work because the `WORKDIR` is specified in the base image.